.github/workflows/push-tests.yml

```diff
@@ -1,13 +1,12 @@
 name: Push Tests
 
 on:
-  pull_request:
-    branches:
-      - "*"
```

Comment:
this caused push tests to be run previously. Now we're only triggering on push to main and in the rdev update script. So these push tests will not be run _as quickly as possible_.

Reply:
When else should it running?

Reply:
Why would we move to running push tests _later_, in the rdev, as opposed to what we're doing now? Can't we push a result to some location/env var that the rdev update job can read and use to determine its flow?

Reply:
It's not later. It runs at the exact same time. The only difference is it's now part of the rdev workflow and deploying rdev.
<img width="755" alt="Screenshot 2024-02-16 at 2 03 07 PM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/1429913/327cfa6b-ff79-4c96-a250-ab0dc9888910">

Reply:
I see what you're saying. Great!